enum/waermenutzung/waermenutzung.go

```diff
@@ -0,0 +1,13 @@
+package waermenutzung
+
+// Stromverbrauchsart/Wärmenutzung Marktlokation
+
+//go:generate stringer --type Waermenutzung
+//go:generate jsonenums --type Waermenutzung
+type Waermenutzung int
+
+const (
```

Comment:
I checked that it's consistent with https://github.com/Hochfrequenz/BO4E-dotnet/blob/6b79477cb6799d71caf3eb2de54d9a8449ee30fc/BO4E/ENUM/Waermenutzung.cs#L8